applications/fax/src/fax_worker.erl

```diff
@@ -908,6 +920,41 @@ convert_openoffice_document(CT, TmpDir, JobId, RespContent) ->
             {'error', <<"can not convert file, try uploading a tiff">>}
     end.
 
+-spec convert_msoffice_document(kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary()) ->
+                                         {'ok', kz_term:ne_binary()} |
+                                         {'error', kz_term:ne_binary()}.
+convert_msoffice_document(CT, TmpDir, JobId, RespContent, Secret) ->
+    Extension = kz_mime:to_extension(CT),
+    FileName = list_to_binary([JobId, ".", Extension]),
+    RequestURL = list_to_binary([?CONVERTAPI_URL, "/", Extension, "/to/tiff?ImageResolutionH=204&ImageResolutionV=392&ScaleImage=true&ScaleProportions=true&ScaleIfLarger=true&ImageHeight=4312&ImageWidth=1728&Secret=", Secret]),
+    Boundary = kz_term:to_hex_binary(crypto:strong_rand_bytes(24)),
+    ContentTypeHeaderValue = list_to_binary(["multipart/form-data; boundary=", Boundary]),
+    Headers = [{"Content-Type", ContentTypeHeaderValue}],
+    Body = list_to_binary(["--", Boundary
+                          ,"\r\nContent-Disposition: form-data; name=\"File\"; filename=\"", FileName, "\""
+                          ,"\r\nContent-Type: ", CT
+                          ,"\r\n\r\n", RespContent
+                          ,"\r\n--", Boundary, "--\r\n"]),
+
+    lager:debug("attemting to convert document using convertapi for JobId: ~s", [JobId]),
+    case kz_http:post(kz_term:to_list(RequestURL), Headers, Body) of
+        {'ok', 200, _RespHeaders, RespBody} ->
+            JObj = kz_json:decode(RespBody),
+            case kz_json:is_defined(<<"Files">>, JObj) of
+                'true' ->
+                    lager:debug("jobid ~s converted successfully using convertapi", [JobId]),
+                    [File|_] = kz_json:get_list_value(<<"Files">>, JObj),
+                    FileDataBase64 = kz_json:get_value(<<"FileData">>, File),
+		    prepare_contents(<<"image/tiff">>, JobId, base64:decode(FileDataBase64), TmpDir);
```

Comment:
Resulted ConverAPI TIFF file is have RGB color space and need to prepare to before passing to FreeSwitch/SpanDSP.
Here is simple call other content converted and pass resulted ConverAPI TIFF file.